CHANGELOG.md

```diff
@@ -1,4 +1,61 @@
+* **New Resource:** `google_iam_principal_access_boundary_policy` ([#8634](https://github.com/hashicorp/terraform-provider-google-beta/pull/8634))
+* **New Resource:** `google_network_management_vpc_flow_logs_config` ([#8623](https://github.com/hashicorp/terraform-provider-google-beta/pull/8623))
+
+IMPROVEMENTS:
+* apigee: added in-place update support for `google_apigee_env_references` ([#8621](https://github.com/hashicorp/terraform-provider-google-beta/pull/8621))
+* apigee: added in-place update support for `google_apigee_environment` resource ([#8627](https://github.com/hashicorp/terraform-provider-google-beta/pull/8627))
+* backupdr: added `ignore_inactive_datasources` and `ignore_backup_plan_references` fields to `google_backup_dr_backup_vault` resource ([#8616](https://github.com/hashicorp/terraform-provider-google-beta/pull/8616))
+* bigquery: added `external_catalog_dataset_options` fields to `google_bigquery_dataset` resource ([#8558](https://github.com/hashicorp/terraform-provider-google-beta/pull/8558))
+* cloudrunv2: added `gcs.mount_options` to `google_cloud_run_v2_service` and `google_cloud_run_v2_job` ([#8613](https://github.com/hashicorp/terraform-provider-google-beta/pull/8613))
+* compute: added `rules` property to `google_compute_region_security_policy` resource ([#8574](https://github.com/hashicorp/terraform-provider-google-beta/pull/8574))
+* compute: added `disks` field to `google_compute_node_template` resource ([#8620](https://github.com/hashicorp/terraform-provider-google-beta/pull/8620))
+* compute: added `replica_names` field to `sql_database_instance` resource ([#8637](https://github.com/hashicorp/terraform-provider-google-beta/pull/8637))
+* compute: added new field `instance_flexibility_policy` to resource `google_compute_region_instance_group_manager` ([#8581](https://github.com/hashicorp/terraform-provider-google-beta/pull/8581))
+* compute: increased `google_compute_security_policy` timeouts from 20 minutes to 30 minutes ([#8589](https://github.com/hashicorp/terraform-provider-google-beta/pull/8589))
+* container: added `control_plane_endpoints_config` field to `google_container_cluster` resource. ([#8630](https://github.com/hashicorp/terraform-provider-google-beta/pull/8630))
+* container: added `parallelstore_csi_driver_config` field to `google_container_cluster` resource. ([#8607](https://github.com/hashicorp/terraform-provider-google-beta/pull/8607))
+* container: added `user_managed_keys_config` field to `google_container_cluster` resource. ([#8562](https://github.com/hashicorp/terraform-provider-google-beta/pull/8562))
+* firestore: allowed single field indexes to support `__name__ DESC` indexes in `google_firestore_index` resources ([#8576](https://github.com/hashicorp/terraform-provider-google-beta/pull/8576))
+* privateca: added support for sub-CA to be activated into STAGED state ([#8560](https://github.com/hashicorp/terraform-provider-google-beta/pull/8560))
```

Comment:
Might be worth updating this entry to mention `google_privateca_certificate_authority`, but I'm not sure what good wording would be. Maybe something like this?

> * privateca: added support for `google_privateca_certificate_authority` with type = "SUBORDINATE" to be activated into "STAGED" state ([#8560](https://github.com/hashicorp/terraform-provider-google-beta/pull/8560))

Reply:
Looks good. Updated. Thanks!